frontend/app/dashboard/src/views/dashboard/webshop/edit/products/ProductPriceBox.vue

```diff
@@ -98,21 +104,45 @@
 </template>
 
 <script lang="ts" setup>
-import { AutoEncoderPatchType } from '@simonbackx/simple-encoding';
-import { Checkbox, ErrorBox, NumberInput, PriceInput, STInputBox, STList, STListItem, Dropdown, Toast, CenteredMessage, useFeatureFlag } from '@stamhoofd/components';
+import { AutoEncoderPatchType, PartialWithoutMethods } from '@simonbackx/simple-encoding';
+import { Checkbox, ErrorBox, NumberInput, PriceInput, STInputBox, STList, STListItem, Dropdown, Toast, useFeatureFlag, LoadingInputBox, NavigationActions } from '@stamhoofd/components';
 import { Product, ProductPrice } from '@stamhoofd/structures';
-import { computed } from 'vue';
+import { computed, onMounted, ref, watch } from 'vue';
+import { useGetOfficialUitpasSocialTariff } from '@stamhoofd/components/src/context/hooks/useGetOfficialUitpasSocialTariff';
+import { useGoToUitpasConfiguration } from '@stamhoofd/components/src/context/hooks/useGoToUitpasConfiguration';
+
+const { getOfficialUitpasSocialTariff } = useGetOfficialUitpasSocialTariff();
 
 const props = defineProps<{
     errorBox: ErrorBox | null;
     productPrice: ProductPrice;
     product: Product;
+    isNew: boolean;
 }>();
 
 const emits = defineEmits<{ (e: 'patch', patch: AutoEncoderPatchType<Product>): void }>();
 const uitpasFeature = useFeatureFlag()('uitpas');
 
 const patchedProductPrice = computed(() => props.productPrice);
+const patchedProduct = computed(() => props.product);
+function addProductPatch(patch: PartialWithoutMethods<AutoEncoderPatchType<Product>>) {
+    emits('patch', patch as AutoEncoderPatchType<Product>);
+}
```

Comment:
Deze types kloppen niet. Je kan PartialWithoutMethods niet casten naar `AutoEncoderPatchType`.


```suggestion
function addProductPatch(patch: PartialWithoutMethods<AutoEncoderPatchType<Product>>) {
    emits('patch', Product.patch({id: props.product.id}).patch(patch));
}
```